src/utils/prim-lexing.fs

```diff
@@ -221,7 +219,9 @@ namespace Internal.Utilities.Text.Lexing
            with get() = endPos
            and  set b =  endPos <- b
 
-        member lexbuf.Lexeme         = Array.sub buffer bufferScanStart lexemeLength
+        member lexbuf.Lexeme         = new System.ReadOnlySpan<'Char>(buffer, bufferScanStart, lexemeLength)
```

Comment:
This most definitely makes sense. This will help get rid of a ton of copying.

Reply:
Great use case of Span.

Reply:
Yeah this is nice. BTW the `new` isn't needed here

Reply:
We should probably rename `lexbuf.Lexeme` to `lexbuf.LexemeView` - as the underlying `Span` could have its values changed. In addition, we should add a comment about it.

Reply:
ok, will change name